lib/tasks/test-sass.js

```diff
@@ -18,17 +18,18 @@ const trueTest = function(config, task) {
 		const componentBase = path.join(__dirname, '../../');
 		const sassFile = path.join(__dirname, 'index.test.scss');
 		const sassTrue = require('${require.resolve("sass-true")}');
+		const sass = require('${require.resolve("sass")}');
```

Comment:
Could we use @financial-times/sass instead? That way we don't need to include two different versions of dart-sass

Reply:
[Unfortunately not without changes to True](https://github.com/oddbird/true/blob/ba0c683c0d976a1f82e94c70fb7299509dcdcc01/lib/main.js#L53).